services/api-server/tests/unit/api_studies/test_api_routes_studies.py

```diff
@@ -111,15 +113,41 @@ async def test_studies_read_workflow(
     assert f"{inexistent_study_id}" in error.errors[0]
 
 
+@pytest.mark.parametrize("capture", ["get_list_study_ports_found.json"])
 async def test_list_study_ports(
     client: httpx.AsyncClient,
     auth: httpx.BasicAuth,
     mocked_webserver_service_api_base: MockRouter,
+    respx_mock_from_capture: Callable[
+        [list[respx.MockRouter], Path, list[SideEffectCallback] | None],
+        list[respx.MockRouter],
+    ],
     fake_study_ports: list[dict[str, Any]],
     study_id: StudyID,
+    project_tests_dir: Path,
+    capture: str,
 ):
     # Mocks /projects/{*}/metadata/ports
 
+    def _get_list_metadata_ports_side_effect(
+        request: httpx.Request,
+        path_params: dict[str, Any],
+        capture: HttpApiCallCaptureModel,
+    ) -> Any:
+        response = capture.response_body
+        assert isinstance(response, dict)
+        if data := response.get("data"):
+            print(data)
+            assert isinstance(data, list)
+            assert len(data) == 5
+        return response
+
+    respx_mock = respx_mock_from_capture(
+        [mocked_webserver_service_api_base],
+        project_tests_dir / "mocks" / capture,
+        [_get_list_metadata_ports_side_effect],
+    )
+
     mocked_webserver_service_api_base.get(
         path__regex=r"/projects/(?P<project_id>[\w-]+)/metadata/ports$",
         name="list_project_metadata_ports",
```

Comment:
Do you still need to mock this `get` method?

Reply:
Sorry, forgot to remove that one. Obviously not.